client/src/components/profile/components/time-line.tsx

```diff
@@ -165,62 +165,60 @@ function TimelineInner({
-          <Modal
-            aria-labelledby='contained-modal-title'
-            onHide={closeSolution}
-            show={solutionOpen}
-          >
-            <Modal.Header closeButton={true}>
-              <Modal.Title id='contained-modal-title' className='text-center'>
-                {`${username}'s Solution to ${
-                  idToNameMap.get(id)?.challengeTitle ?? ''
-                }`}
-              </Modal.Title>
-            </Modal.Header>
-            <Modal.Body>
-              <SolutionViewer
-                challengeFiles={challengeData.challengeFiles}
-                solution={challengeData.solution ?? ''}
-              />
-            </Modal.Body>
-            <Modal.Footer>
-              <Button onClick={closeSolution}>{t('buttons.close')}</Button>
-            </Modal.Footer>
-          </Modal>
-        )}
-        {totalPages > 1 && (
-          <TimelinePagination
-            firstPage={firstPage}
-            lastPage={lastPage}
-            nextPage={nextPage}
-            pageNo={pageNo}
-            prevPage={prevPage}
-            totalPages={totalPages}
-          />
-        )}
-      </FullWidthRow>
+      <h2 className='text-center'>{t('profile.timeline')}</h2>
+      {completedMap.length === 0 ? (
+        <p className='text-center'>
+          {t('profile.none-completed')}&nbsp;
+          <Link to='/learn'>{t('profile.get-started')}</Link>
+        </p>
+      ) : (
+        <Table condensed={true} striped={true}>
+          <thead>
+            <tr>
+              <th>{t('profile.challenge')}</th>
+              <th>{t('settings.labels.solution')}</th>
+              <th className='text-center'>{t('profile.completed')}</th>
+            </tr>
+          </thead>
+          <tbody>
+            {sortedTimeline.slice(startIndex, endIndex).map(renderCompletion)}
+          </tbody>
+        </Table>
+      )}
+      {id && (
+        <Modal
+          aria-labelledby='contained-modal-title'
+          onHide={closeSolution}
+          show={solutionOpen}
+        >
+          <Modal.Header closeButton={true}>
+            <Modal.Title id='contained-modal-title' className='text-center'>
+              {`${username}'s Solution to ${
+                idToNameMap.get(id)?.challengeTitle ?? ''
+              }`}
+            </Modal.Title>
+          </Modal.Header>
+          <Modal.Body>
+            <SolutionViewer
+              challengeFiles={challengeData.challengeFiles}
+              solution={challengeData.solution ?? ''}
+            />
+          </Modal.Body>
+          <Modal.Footer>
+            <Button onClick={closeSolution}>{t('buttons.close')}</Button>
+          </Modal.Footer>
+        </Modal>
+      )}
+      {totalPages > 1 && (
+        <TimelinePagination
+          firstPage={firstPage}
+          lastPage={lastPage}
+          nextPage={nextPage}
+          pageNo={pageNo}
+          prevPage={prevPage}
+          totalPages={totalPages}
+        />
+      )}
```

Comment:
> there should be elements for "Project name", "Solution", and "Completed"

How did you address this, btw? because I think I am misunderstanding it as adding `<span class='sr-only'>Project name</span>`. :thinking:

Reply:
The only thing I did in this file is increase the table width as all three columns were already there line:184. 
You can check `renderCompletion` function.

Reply:
So I was misunderstanding, thank you for conforming

https://github.com/freeCodeCamp/freeCodeCamp/blob/898a9331a6b5889f2a4122f379b2022a73aae4ea/client/src/components/profile/components/time-line.tsx#L118-L150=plain?1